pages/api/collectionindex/[collection].ts

```diff
@@ -60,19 +65,34 @@ export default async function (req: NextApiRequest, res: NextApiResponse) {
     if (nftIndex === undefined) {
         const highestReservedIndex = await prisma.collectionIndexes.findFirst({
             where: {
-                collectionId: dbCollection.id
+                AND: [
```

Comment:
remove this - we don't care, we want highest index, it can be submitted, or free